src/server/setting-controller.js

```diff
@@ -37,7 +39,7 @@ class SettingController {
   }
 
   registerApi(app) {
-    app.post('/api/v1/settings', [needsPermission(permissions.EDIT_SETTINGS), jsonParser], async (req, res) => {
+    app.post('/api/v1/settings', [needsPermission(permissions.EDIT_SETTINGS), jsonParser, validateBody(saveSettingsBodySchema)], async (req, res) => {
       const { settings } = req.body;
       await this.settingService.saveSettings(settings);
       return res.send({ settings });
```

Comment:
Will we no longer merge the `setting-controller` and `settings-controller` into one module?

Reply:
I completely overlooked that. They're merged now. Thanks!